lib/css/CssLoadingRuntimeModule.js

```diff
@@ -345,11 +345,11 @@ class CssLoadingRuntimeModule extends RuntimeModule {
 														'if(event.type !== "load") {',
 														Template.indent([
 															"var errorType = event && event.type;",
-															"var realSrc = event && event.target && event.target.src;",
```

Comment:
We don't have `src` on the `link` tag, we use `href` to provide path to CSS file